app/controllers/reservations_controller.rb

```diff
@@ -1,3 +1,23 @@
-def my_reservations
-  @reservations = current_user.reservations
+# app/controllers/reservations_controller.rb
+class ReservationsController < ApplicationController
+  before_action :authenticate_user!
+
```

Comment:
authenticate_user! belong to devise gem. In this case, we have to find another way to get the user ID cause what we implement is a simple login page with only the username.